packages/botonic-react/tests/webchat/buttons-disabler.test.jsx

```diff
@@ -163,15 +226,15 @@ describe('TEST: ButtonsDisabler (Disabling buttons in Webchat)', () => {
       'url',
       'target',
       'webview',
-      'disabled',
+      // 'disabled',
       'children',
     ])
     expect(Object.keys(element2ButtonProps)).toEqual([
       'payload',
       'url',
       'target',
       'webview',
-      'disabled',
+      // 'disabled',
```

Comment:
should this be commented or it can be removed?

Reply:
removed